app/components/language-select.js

```diff
@@ -0,0 +1,39 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  config: Ember.inject.service(),
+  i18n: Ember.inject.service(),
+  languageOptions: Ember.computed(function() {
+    return Object.keys(this.get('languageMap'));
+  }),
+  onFinish: null,
+  languageMap: Ember.computed(() => {
```

Comment:
Also, I added localized English/Spanish/French values for the labels "English", "Spanish", and "French", but stopped short of adding the respective keys for all supported languages (i.e. the Russian translation for the label "Urdu", etc.).